### PR TITLE
Fix mis-alignment for answer dragging sorter

### PR DESCRIPTION
**Pull request by nikolas**

### CODE REVIEW

File: quizblock/templates/quizblock/quiz_detail.html [hunk 1]
```diff
@@ -15,8 +15,8 @@
             padding-left: 1.5em;
             background-color: #ddd;
         }
-        #questions li span { position: absolute; margin-left: -1.3em; }
-        .dragging {border: #f60 solid 1px;}
+        #questions li .draghandle { cursor: move; }
```
Comment (sdreher) [inline quizblock/templates/quizblock/quiz_detail.html]: Does quiz_detail.html have a draghandle? I thought that was more for editing?

Comment (nikolas) [inline quizblock/templates/quizblock/quiz_detail.html]: Oh right. I'll remove this part.

Comment (nikolas) [inline quizblock/templates/quizblock/quiz_detail.html]: quiz_detail.html does initialize a jquery sortable widget, actually.

Comment (sdreher) [inline quizblock/templates/quizblock/quiz_detail.html]: ah, right, my mistake. The "view" is quizblock.html and then the edit detail view is "quiz_detail" per CBV conventions.

**Pull request merged**